### ISSUE TITLE
Auth

### ISSUE THREAD

**Issue by jgp3199** [Cog-Creators/Cog-Board#100]
## Description of cog: 
When a user joins the server they are only allowed to see a certain channel(s)... Upon joining the bot sends them a message answering a simple question, for example a math question like 2+2 and when the user give the correct answer they are allowed to see the entire discord.... (Basically a special cog designed to prevent raids that i happen to like the idea of)

## API's or other information for cog:

<APIs and integrations here>

## Other info (Bounty, issues, ETC): 

$10-$20 USD

**Comment by calebj** [comment]
I already did a similar "captcha" cog for another user. I can get you in touch with them after work. It doesn't operate through adding a role and then removing it; rather, it adds an "approved" role. That said, the logic can be inverted easily enough.

**Comment by jgp3199** [comment]
Sweet much appreciated...

**Comment by Aioxas** [comment]
Would like an update on the situation of this cog, is the commission complete, stale, or?

**Comment by calebj** [comment]
The original person disappeared or changed their username, I can't find them anymore. It's been a while. Are you still interested, or should I just publish a captcha cog for general use?

**Comment by calebj** [comment]
Yeah, I'm just gonna publish a generalized captcha cog. It'll be done soon™.
